### ISSUE TITLE
New 5th Pattern Solution

### ISSUE THREAD

**Issue by skateron** [mansuf/zippyshare-downloader#15]
Hi,

I use my own personal Bash implementation of a Zippyshare downloader, recently it broke, I hate when that happens. Thanks to the debug mode's Inspector of Firefox, I figured it out, despite not being a programmer. I like your project, but my Python skills are too bad to contribute, and I hope the solution of mine will help to bring back your script too.

Short: "+ a() + b() + c() + d + 5/5" is "+ 11" in reality.

```
<span id="omg" class="2" style="display:none;"></span>
<script type="text/javascript">
    var a = function() {return 1};
    var b = function() {return a() + 1};
    var c = function() {return b() + 1};
    var d = document.getElementById('omg').getAttribute('class');
    if (true) { d = d*2;}
    document.getElementById('dlbutton').href = "/d/XXXXXXX/"+(981255%1000 + a() + b() + c() + d + 5/5)+"/example.txt";
    if (document.getElementById('fimage')) {
        document.getElementById('fimage').href = "/i/XXXXXXX/"+(981255%1000 + a() + b() + c() + d + 5/5)+"/example.txt";
    }
</script>
```

The math is, simply:
981255%1000+11=266

**Comment by skateron** [comment]
Sorry y'all, I edited the code to include the correct 981255 in it. Have a nice day!

**Comment by skateron** [comment]
For future changes, I should note Github-based [Arion-Kun's VariantLog](https://github.com/Arion-Kun/GoZippy/tree/main/VariantLog) which keeps samples to work with.

**Comment by mansuf** [comment]
> I hope the solution of mine will help to bring back your script too.

Thank you :)

Recently i released 4th pattern, but zippyshare changed their code again.

**Comment by mansuf** [comment]
Shipped to v0.3.3 ! 😀
